rollup.config.js

```diff
@@ -40,7 +40,8 @@ module.exports = [
 		input,
 		plugins: [
 			inject({
-				ResizeObserver: 'resize-observer-polyfill'
+				ResizeObserver: 'resize-observer-polyfill',
+				Promise: 'promise-polyfill'
```

Comment:
any idea how much bigger this makes the output?

Reply:
About 3kb